package.json

```
@@ -67,8 +67,8 @@
     "test:depcheck": "yarn depcheck",
     "test:tgz-check": "./scripts/tgz-check.sh",
     "test:attribution-check": "./scripts/attributions-check.sh",
-    "test:merge-coverage": "nyc merge ./tests/coverage ./tests/merged-coverage/coverage.json",
-    "test:validate-coverage": "nyc check-coverage --nycrc-path ./coverage-thresholds.json -t ./tests/merged-coverage/",
+    "test:merge-coverage": "nyc report --temp-dir ./tests/coverage --report-dir ./tests/merged-coverage/ --reporter json --reporter text --reporter lcovonly",
+    "test:validate-coverage": "nyc check-coverage --nycrc-path ./coverage-thresholds.json -t ./tests/merged-coverage/ --reporter json --reporter text --reporter lcov",
```

Looks like the flags were accidentally left on this command as well:
```suggestion
    "test:validate-coverage": "nyc check-coverage --nycrc-path ./coverage-thresholds.json -t ./tests/merged-coverage/",
```

It doesn't seem to affect the command, so I guess they are ignored

---

CodeCov pulls them in but it isn't impacting the GH Action. The path issue should be sorted out and I will update the yarn command after that.